superset/assets/src/SqlLab/main.less

```diff
@@ -321,10 +321,9 @@ div.Workspace {
 
 div.tablePopover {
   opacity: 0.7 !important;
-}
-
-div.tablePopover:hover {
-  opacity: 1 !important;
+  &:hover {
```

Comment:
this is a styling thing, but personally i prefer a new line between normal styles and the `&:foo {` line. I think there's a way to lint for that, if so maybe you could add that rule in another PR. if not, maybe we could start standardizing that here if you agree?

Reply:
I prefer that too, but glad you caught ones I missed. I'll look for a linting rule.